modules/connatixBidAdapter.js

```diff
@@ -53,21 +72,28 @@ export const spec = {
    */
   isBidRequestValid: (bid = {}) => {
     const bidId = deepAccess(bid, 'bidId');
-    const mediaTypes = deepAccess(bid, 'mediaTypes', {});
+    const mediaTypes = deepAccess(bid, 'mediaTypes', null);
```

Comment:
why did you change the 3rd param from {} to null?

Reply:
The bid request should be invalidated if the `mediaTypes` is missing. I wanted the default value to be null (3rd param), so I can check bellow if its actually missing or not.

Reply:
nvm, changed back:))